Mathlib/Tactic/ScopedNS.lean

```diff
@@ -18,7 +18,7 @@ open Lean
 `scoped[NS]` is similar to the `scoped` modifier on attributes and notations,
 but it scopes the syntax in the specified namespace instead of the current namespace.
 ```
-scoped[Matrix] infixl:72 " ⬝ᵥ " => Matrix.dotProduct
+scoped[Matrix] infixl:72 " ⬝ᵥ " => dotProduct
```

Comment:
Should this be a new example of notation that remains scoped, e.g. `Matrix.transpose`?

Reply:
What is your suggestion please?

Reply:
Ah, sorry, I overlooked which file you were commenting on.

Reply:
Thanks a lot for your patience and quick replies!